src/vs/workbench/contrib/chat/browser/chatSessions.contribution.ts

```diff
@@ -1083,6 +1083,10 @@ export class ChatSessionsService extends Disposable implements IChatSessionsServ
 	public getContentProviderSchemes(): string[] {
 		return Array.from(this._contentProviders.keys());
 	}
+
+	public isChatSessionInProgressStatus(state: ChatSessionStatus): boolean {
```

Comment:
personally I would tend to make this a free-floating function so that people don't need the whole service just to check the status. I like having a namespace so I get pseudo-methods on the enums like `ChatSessionStatus.isInProgress(state)`

https://github.com/microsoft/vscode/blob/17cf1ce11ac873561637f74dadb80dc84a4255d2/src/vs/workbench/contrib/mcp/common/mcpTypes.ts#L576-L588